### PR TITLE
Added my name to the job portal list

### PR DESCRIPTION
**Pull request by mailsg**
Hi Team,

I have added my name to the job portal list. @dga 

Thanks.

### CODE REVIEW

File: Submissions/@mailsg.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Sandeep Ghosh",
+  "img": "https://avatars.githubusercontent.com/u/105475440?v=4",
+  "email": "mailsg0912@gmail.com",
+  "links": {
+    "website": " ",
```
Comment (ProsperousHeart) [inline Submissions/@mailsg.json:6]: please remove the space

File: Submissions/@mailsg.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Sandeep Ghosh",
+  "img": "https://avatars.githubusercontent.com/u/105475440?v=4",
+  "email": "mailsg0912@gmail.com",
+  "links": {
+    "website": " ",
+    "linkedin": "https://www.linkedin.com/in/sandeep-ghosh-660830166/",
+    "github": "https://github.com/mailsg"
+  },
+  "jobTitle": "SRE/SDE",
```
Comment (ProsperousHeart) [inline Submissions/@mailsg.json:10]: we could leave this title, but just making a suggestion to spell them out

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]

**Comment by mailsg** [comment]
Hi,
I have made the changes as suggested. 
1. Removed the space from the website.
2. Defined the title.

Thanks.

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
everything looks good except for the @ in the file name - once that is removed this can be approved

**Review by ProsperousHeart — APPROVED** [review]
Approved! Thank you

**Pull request merged**